examples/websocket_server/WebSocketServer.cc

```diff
@@ -14,6 +14,7 @@ class WebSocketChat : public drogon::WebSocketController<WebSocketChat>
                              const WebSocketConnectionPtr &) override;
     WS_PATH_LIST_BEGIN
     WS_PATH_ADD("/chat", Get);
+    WS_ADD_PATH_VIA_REGEX("/[^/]*", Get);
```

Comment:
@an-tao the naming feels strange on the macro, I say we make it have the same prefix as the original and add `"_VIA_REGEX"` to it:
```c++
WS_PATH_ADD_VIA_REGEX
```

What do you say?

Reply:
What about `WS_PATH_REGEX_ADD`? This corresponds to the style of `WS_PATH_LIST_BEGIN` as well, i.e., `WS_` + Noun + Verb.

Reply:
I use the same format as the http using
<img width="808" alt="{6E310ABF-9D6D-4890-8237-33A8616C4234}" src="https://github.com/drogonframework/drogon/assets/91441995/e6513546-5ec5-4fe5-869a-fa085f9a8c72">

Reply:
Damn, and making naming changes to macros will break compatibility.

We can sick to one standard naming convention, and keep the old ones for compatibility purposes.

My suggestion is to use scoped naming convention:
```c++
PATH_ADD
PATH_ADD_VIA_REGEX

METHOD_ADD
METHOD_ADD_TO
METHOD_ADD_VIA_REGEX

WS_ADD
WS_ADD_VIA_REGEX
```